examples/platform/qpg6100/project_include/CHIPProjectConfig.h

```diff
@@ -131,3 +131,14 @@
  * Disable Full Thread Device features
  */
 #define CHIP_DEVICE_CONFIG_THREAD_FTD 0
+
+/**
+ *  @def CHIP_CONFIG_MAX_DEVICE_ADMINS
+ *
+ *  @brief
+ *    Maximum number of administrators that can provision the device. Each admin
+ *    can provision the device with their unique operational credentials and manage
+ *    their access control lists.
+ */
+#define CHIP_CONFIG_MAX_DEVICE_ADMINS 1
```

Comment:
I thought the number we were looking at was 3-5?

Reply:
Is there a spec minimum we need to develop with at this point ?

Reply:
After discussion on Slack - set to 4 - a minimum of 3 + 1 to rotate credentials.